plugins/out_stackdriver/stackdriver.c

```diff
@@ -2491,6 +2491,16 @@ static void cb_stackdriver_flush(struct flb_event_chunk *event_chunk,
     if (ret == 0) {
         update_http_metrics(ctx, event_chunk, ts, c->resp.status);
     }
+
+    /* Update retry count if necessary */
+    if (ret_code == FLB_RETRY) {
+        char tmp[32];
```

Comment:
Variables need to be declared at the beginning of the function. https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#variable-definitions